tests/integration/tickstore/test_tickstore.py

```diff
@@ -0,0 +1,8 @@
+from arctic.arctic import Arctic
+
+
+def test_stringify_tickstore_library(mongo_host):
```

Comment:
Can we make this a unit test? Spinning up a mongodb and creating libraries, etc. is inefficient just to call `str(...)` :)